web/README.md

```diff
@@ -0,0 +1,37 @@
+# Drand Web App
+
+## Requirements
+
+- [hugo](https://gohugo.io)
+
+## Testing
+
+In order to test, we have to start a fake drand server to get the randomness from. To so such, run:
+```
+python3 server_example/script.py
+```
+
+The public key, previous and randomness fields are generated thanks to the `server_example/main.go` file.
+
+## Production
+
+When production ready, the folder `server_example` can be removed.
```

Comment:
We don't HAVE to remove it, do we ? Or what happens if we dont ?

Reply:
not a must do, just won't be useful anymore

Reply:
then we can simplify the readme by removing this line. It's one instruction less :  one chance of doing stg wrong less ;)